src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
We're in ArrayList here... we shouldn't need to spell out the full System.Collections.ArrayList name... can we change the tool to do that?  I'd expect Roslyn's auto-simplify support would help here.  Basically we should use the shortest names possible based on what namespaces/etc. are imported.